pkg/entities/emojis.go

```diff
@@ -0,0 +1,33 @@
+package entities
+
+import (
+	"gorm.io/gorm"
+
+	"github.com/defipod/mochi/pkg/logger"
+	"github.com/defipod/mochi/pkg/model"
+	"github.com/defipod/mochi/pkg/model/errors"
+)
+
+func (e *Entity) GetListEmojis(listCode []string) ([]*model.Emojis, error) {
+	emojis, err := e.repo.Emojis.GetListEmojis(listCode)
+	if err != nil {
+		e.log.Fields(logger.Fields{}).Error(err, "[entity.GetListEmojis] repo.emojis.GetListEmojis failed")
```

Comment:
Maybe we do not need Fields here 
```suggestion
		e.log.Error(err, "[entity.GetListEmojis] repo.emojis.GetListEmojis failed")
```